src/components/FieldHelp/index.js

```diff
@@ -69,7 +69,7 @@ FieldHelp.propTypes = {
   tooltip: PropTypes.string, // tooltip shown when hovering the text
   children: PropTypes.any,
   container: PropTypes.any,
-  getParentPosition: PropTypes.func,
+  //  getParentPosition: PropTypes.func,
```

Comment:
How about just dropping these commented proptypes? Why are they left commented?

Applies to `PageRouter.js`, `Pages/index.js` as well.

Reply:
done